external/radiation/radiation/config.py

```diff
@@ -86,9 +90,11 @@ class GFSPhysicsControl:
     )
     do_only_clearsky_rad: bool = False
     swhtr: bool = True
-    lprnt: bool = False
     lwhtr: bool = True
+    lprnt: bool = False
     lssav: bool = True
+    nsswr: Optional[int] = None
+    nslwr: Optional[int] = None
```

Comment:
what's the expected behavior if these are not set? Would a default of 1 make more sense?

Reply:
This value has to be set before running the radiation port and is not actually configurable since it will be overwritten as a function of `fhswr/fhlwr` and the physics timestep, so a default value would never be used. I think there's some conflation of immutable namelist flags and mutable control data structure going on here, but I would prefer to deal with that in a future PR that refactors the rad wrapper a bit.